app/services/route-context.ts

```diff
@@ -0,0 +1,68 @@
+import { alias } from '@ember-decorators/object/computed';
+import { service } from '@ember-decorators/service';
+import Service from '@ember/service';
+import { task, TaskInstance } from 'ember-concurrency';
+import DS from 'ember-data';
+
+import Guid, { ReferentModel, ReferentModelName } from 'ember-osf-web/models/guid';
+
+/**
+ * Holds route information that might be needed by engines, e.g. the current guid
+ * at the root of the route's path and the object it refers to.
+ */
+export default class RouteContextService extends Service {
+    @service store!: DS.Store;
+
+    guid?: string;
+    modelName?: ReferentModelName;
+    guidTaskInstance?: TaskInstance<ReferentModel>;
+
+    @alias('guidTaskInstance.last.value')
+    model?: ReferentModel;
```

Comment:
Right. Though if it were it wouldn't work, because the alias is wrong. Should be either `loadModel.last.value` or `guidTaskInstance.value`. (`loading` is also wrong...)

I think I'll just delete these two aliases, actually. The TaskInstance API should be comfortable to use everywhere, these don't actually add value.